__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -2930,4 +2930,15 @@ describe('storage methods', () => {
     await expect(mshWallet.getAccessData()).resolves.toEqual(mshAccessData);
     await expect(mshWallet.getMultisigData()).resolves.toEqual(mshAccessData.multisigData);
   });
+
+  it('should return if the wallet is a hardware wallet', async () => {
```

Comment:
Do we have a method in the integration tests to create a readonly wallet (from an xpub)? I prefer creating the wallet with the normal process and then check if it saved correctly the hardware flag, instead of manually change the access data

Reply:
Created a method to generate a readonly wallet from the pre-calculated wallets.
This way the generated wallet will come with the readonly flag and optionally we can enable the hardware flag to use in this test.